docs/source/geo_def.rst

```diff
@@ -181,19 +255,19 @@ Two swaths can be concatenated if their coloumn count matches
  >>> lons2 = np.ones((300, 20))
  >>> lats2 = np.ones((300, 20))
  >>> swath_def2 = geometry.SwathDefinition(lons=lons2, lats=lats2)
- >>> swath_def3 = swath_def1.concatenate(swath_def2) 
- 
+ >>> swath_def3 = swath_def1.concatenate(swath_def2)
+
 Geographic coordinates and boundaries
 -------------------------------------
 A ***definition** object allows for retrieval of geographic coordinates using array slicing (slice stepping is currently not supported).
 
-All ***definition** objects exposes the coordinates **lons**, **lats** and **cartesian_coords**. 
-AreaDefinition exposes the full set of projection coordinates as **projection_x_coords** and **projection_y_coords** 
+All ***definition** objects exposes the coordinates **lons**, **lats** and **cartesian_coords**.
+AreaDefinition exposes the full set of projection coordinates as **projection_x_coords** and **projection_y_coords**
```

Comment:
Just a note: Projection coordinates can also be in degrees, but I think there's no better naming available. And the same naming scheme was already agreed for area extents in the YAML config so this follows the same idea.